src/three-components/ModelScene.js

```diff
@@ -261,6 +264,10 @@ export default class ModelScene extends Scene {
    */
   onModelLoad() {
     this.applyRoomSize();
+
+    // Ensure the model renders after the skysphere
+    this.model.traverse(obj => obj.renderOrder = 1000);
```

Comment:
Since this is guaranteed to happen, why not do it in the loader traversal where materials are cloned?